flexbox/src/main/java/com/google/android/flexbox/FlexboxLayout.java

```diff
@@ -648,7 +648,7 @@ private void measureVertical(int widthMeasureSpec, int heightMeasureSpec) {
                     child.getMeasuredWidth() + lp.leftMargin + lp.rightMargin);
 
             if (isWrapRequired(mFlexWrap, heightMode, heightSize, flexLine.mainSize,
```

Comment:
Likewise you can change the fourth argument as child.getMeasuredHeight() + lp.topMargin + lp.bottomMargin instead of adding a new argument.